monai/networks/nets/vista3d.py

```diff
@@ -78,6 +78,25 @@ def __init__(self, image_encoder: nn.Module, class_head: nn.Module, point_head:
         self.NINF_VALUE = -9999
         self.PINF_VALUE = 9999
 
+    def update_slidingwindow_padding(self, pad_size: list | None, labels: torch.Tensor | None, prev_mask: torch.Tensor | None, point_coords: torch.Tensor | None):
+        """ Image has been padded by sliding window inferer. The related padding need to be performed outside of slidingwindow inferer.
+        Args:
+            pad_size: padding size passed from sliding window inferer.
+            labels: image label ground truth.
+            prev_mask: previous segmentation mask.
+            point_coords: point click coordinates.
+        """
+        if pad_size is None:
+            return labels, prev_mask, point_coords
+        if labels is not None:
+            labels = F.pad(labels, pad=pad_size, mode='constant', val=0)
```

Comment:
Hi @heyufan1995 , could you double confirm the arg `val` here? Torch.nn.function.pad does not have this arg
https://pytorch.org/docs/stable/generated/torch.nn.functional.pad.html
I will push a commit to remove it first

Reply:
oh, it has `value`, I guess that's what you'd like to use, let me rename it